### ISSUE TITLE
Example code in storybook

### ISSUE THREAD

**Issue by arifszn** [atlassian/react-beautiful-dnd#2024]
### Description

There are lots of nice examples in storybook. But I can not find any codes for the examples. Am I missing something? Shouldn't we able to see the codes for the examples?

**Comment by 2stash** [comment]
@arifszn All the code is available, but you actually have to search through repo. I also expected the storybooks to have links to the code, but they do not, or at least I didn't see a button.

All code from the storybook is in the source folder. Search does work if the term is specific, like lazy loading. Otherwise just clicking through and following the code has worked for me.

https://github.com/atlassian/react-beautiful-dnd/tree/master/stories/src

**Comment by arifszn** [comment]
Yes, after much struggle I was able to replicate the demo from src files.

**Comment by arifszn** [comment]
@2stash I can't work the horizontal(x-axis) scroll like the demo. Have you ever faced the problem?

**Comment by 2stash** [comment]
I followed the walkthru demo linked below and had no problem.

https://egghead.io/lessons/react-create-reorderable-horizontal-lists-with-react-beautiful-dnd-direction-prop

**Comment by danieldelcore** [comment]
related https://github.com/atlassian/react-beautiful-dnd/issues/1977

**Comment by danieldelcore** [comment]
I've been thinking about adding [@storybook/addon-storysource](https://www.npmjs.com/package/@storybook/addon-storysource). How would you folks feel about that?

**Comment by 2stash** [comment]
@danieldelcore  That would be awesome! Let me know if you want to offload any of the work. I am trying to get more involved in the open source libraries I use.

**Comment by danieldelcore** [comment]
@2stash no worries at all! If you'd like to give it a go I'd me more than happy to review a pr for you 😀 Otherwise happy to give it a go when I get a chance 😀

**Comment by 2stash** [comment]
@danieldelcore I am starting on it right now!

**Comment by 2stash** [comment]
@danieldelcore Hey a quick update incase you have anything to add. I was not familiar with  storybookjs, so went through their documentation to get up to speed. I was able to add the storysource addon to a new project, however it is not working when I try to add it to this project. My suspicion, which I will explore more tomorrow, is that there is an issue with the revisions. 

What I did: 
- Installed @storybook/addon-storysource 
  "@storybook/addon-storysource": "^6.1.11",
- Updated .storybook/main.js to include '@storybook/addon-storysource'
- (also tried to update csp-server/webpack.config.js with 
```
      {
        test: /.stories.jsx?$/,
        loaders: [require.resolve("@storybook/addon-storysource/loader")],
        test: path.resolve(__dirname, "../stories"),
        enforce: "pre"
      },
```

What happens: 
I can see the stories briefly, the website crashes and I get a bunch of TypeErrors.

What I am going to try next.
I am going to try find which older version of storysource I should use with: 
    "@storybook/react": "^5.3.18",
    "@storybook/theming": "^5.3.18",

In the documentation, the syntax seems to change with importing the addons, as well as I found some issues on stackoverflow related to upgrading the packages.

**Comment by danieldelcore** [comment]
Hey @2stash, no worries at all! It can be a little bit fiddly (I found out the hard way with another repo of mine - ha!). Would you like to put your current work into a PR and we can pair on it?

**Comment by 2stash** [comment]
@danieldelcore Sounds awesome. I opened PR "added storysource 5.3.18 #2040" 

https://github.com/atlassian/react-beautiful-dnd/pull/2040

The storysource was working localing but I am having an issue failing CI. I am failing the yarn-lock needs updating test. I am looking into it. Spoiler alert, this is my first PR :)

**Comment by danieldelcore** [comment]
Great! I'm just away on holidays but I'll have a look when I'm back at work in the new year!

**Comment by gfox1984** [comment]
Any news on this?

**Comment by Eronmmer** [comment]
Any update on this??

**Comment by danieldelcore** [comment]
Hey @gfox1984 @Eronmmer 👋 

We ended up adding [story source](https://storybook.js.org/addons/@storybook/addon-storysource) to our [storybook instance](https://react-beautiful-dnd.netlify.app/?path=/story/single-vertical-list--basic) (press "A" to show addons), but we then realised that the stories themselves were too abstracted to be useful for examples.

We'll need to either rewrite the example code or look at some other alternatives.

**Comment by subwaymatch** [comment]
@danieldelcore Thank you for the tip to display addons! Although the stories may be abstract, being able to see the source code makes the docs at least 10x useful. 👍
